### ISSUE TITLE
Feature request.

### ISSUE THREAD

**Issue by jtannenba** [kantlivelong/OctoPrint-PSUControl#253]
Not a problem, but a request.

Request a button/click or ?  for maintenance so printer won't shut down 
I know we can disable the pluggin, but a button on the title bar would be
convenient.

Thanks

**Comment by jeloneal** [comment]
Great idea! I was just about creating a new issue for the same use case, but had a different idea how to achieve it:

I can only refer to my Prusa Mini, but when doing maintenance like changing filament or calibrating first layer, I normally disconnect the printer from octoprint via printers LCD menu. If I forget to turn off the PSU off function it will turn off the printer.

**My idea was if we could prevent turning the PSU off, when the printer is not connected to octoprint.** What do you think about that?

**Comment by jtannenba** [comment]
I don't know, my printer is always connected to Octoprint. Or do you mean when the printer is offline?  If yes, would be great.

**Comment by jeloneal** [comment]
Yes, by not connected I meant that Octoprint is not connected to the printer. The cable is of course connected all the time though.

**Comment by jtannenba** [comment]
Yep!  Would make maintenance easier.  I've had the PSU turn it off when I was working my Ender3 V2.
